### DISCUSSION TITLE
Previews & Projections

### DISCUSSION THREAD

**Discussion by hugo082** [prisma/prisma#15076] [Q&A] [answered]
Hi, I've seen some Prisma features from Prisma Days videos (like [there](https://youtu.be/9awmjREn040?t=1227)) that I cannot find in [the notion's roadmap](https://prismaio.notion.site/Prisma-Roadmap-50766227b779464ab98899accb98295f). Can you update us on the advancements of these projects? 🙏 

Especially [previews](https://www.youtube.com/watch?v=oojjJh-51ZI) & [projections](https://www.youtube.com/watch?v=yjlCjQy_VP8)

**Comment by austincrim** [comment, marked as answer]
Hey @hugo082 👋,

We don't have any major updates to share at this time, but we would love to hear more about your use cases and what you would like to see from these features.

Would you be interested in [scheduling a quick call](https://calendly.com/alberto-prisma/pdp-45-min-zoom-call) to chat?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
